docs/excel/excel-add-ins-conditional-formatting.md

```diff
@@ -0,0 +1,305 @@
+# Apply conditional formatting to Excel ranges
+
+The Excel JavaScript Library provides APIs to apply conditional formatting to data ranges in your worksheets. This functionality makes large sets of data easy to visually parse. The formatting also dynamically updates based on changes within the range. The following articles provide detailed information about the full conditional formatting capabilities within Excel.
```

Comment:
The opening paragraph should be limited to describing what's in *this* article. Consider moving the final sentence of  and the two bullet points that follow into a [!NOTE] that immediately follows the opening paragraph.